package.json

```diff
@@ -667,6 +676,16 @@
       {
         "command": "espIdf.getEspMdf",
         "title": "%espIdf.getEspMdf.title%"
+      },
+      {
+        "command": "espIdf.jtag_flash",
+        "title": "%espIdf.jtag_flash.title%",
+        "category": "ESP-IDF"
+      },
+      {
+        "command": "espIdf.selectFlashMethodAndFlash",
+        "title": "Select Flash Device and Flash",
```

Comment:
Add Localization for `Select Flash Method` ? 

Should also rename old  flash method to Flash (UART)